tools/internal-rules/multiline-comment-style.js

```diff
@@ -0,0 +1,25 @@
+/**
+ * @fileoverview A modified version of the `multiline-comment-style` rule that ignores banner comments.
+ * @author Teddy Katz
+ */
+
+"use strict";
+
+const ruleComposer = require("eslint-rule-composer");
+const multilineCommentStyle = require("../../lib/rules/multiline-comment-style");
+
+//------------------------------------------------------------------------------
+// Rule Definition
+//------------------------------------------------------------------------------
+
+// The `no-invalid-meta` internal rule has a false positive here.
+// eslint-disable-next-line rulesdir/no-invalid-meta
```

Comment:
unrelated: no-invalid-meta seems ideal to be a rule in `eslint-plugin-eslint-plugin`, so that other plugin authors can use easily.